src/RA_Integration.vcxproj

```diff
@@ -210,11 +223,21 @@
     <CharacterSet>MultiByte</CharacterSet>
     <PlatformToolset>v141_xp</PlatformToolset>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Analysis|Win32'" Label="Configuration">
+    <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <CharacterSet>MultiByte</CharacterSet>
+    <PlatformToolset>v141</PlatformToolset>
```

Comment:
Will not work targeting the XP compatible tool-set as the SAL annotations and runtime libraries between XP and 7 are incompatible.